<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <layout class="QVBoxLayout" name="chatVerticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,2,1">
     <item>
      <widget class="QTextEdit" name="chatTextEdit"/>
     </item>
     <item>
      <widget class="QListWidget" name="friendListWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="deleteFriendBtn">
         <property name="text">
          <string>删除好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshFriendBtn">
         <property name="text">
          <string>刷新好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showOnlineUserBtn">
         <property name="text">
          <string>显示在线用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchUserBtn">
         <property name="text">
          <string>查找用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="privateChatBtn">
         <property name="text">
          <string>私聊</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QLineEdit" name="chatLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
